corehq/apps/callcenter/indicator_sets.py

```diff
@@ -177,7 +177,7 @@ def case_types(self):
         """
         :return: Set of all case types for the domain excluding the CallCenter case type.
         """
-        case_types = set(get_case_types_for_domain(self.domain))
+        case_types = get_case_types_for_domain(self.domain)
```

Comment:
left this using the couch version since the whole of callcenter needs to be re-thought